### PR TITLE
[8.0][FIX] account_financial_report: Fix syntactical error

### PR DESCRIPTION
**Pull request by geluboros**
A syntactical error was introduced during the lint improvements.

### CODE REVIEW

File: account_financial_report/wizard/wizard.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def _check_date(self, cr, uid, data, context=None):
 
         if res:
             if data['form']['date_to'] > res[0]['date_stop'] or\
-               data['form']['date_from'] < res[0]['date_start']):
+               data['form']['date_from'] < res[0]['date_start']:
```
Comment (pedrobaeza) [inline account_financial_report/wizard/wizard.py]: Indent this line one level more

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please change version number according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Comment by geluboros** [comment]
Hi Pedro,

Changes done, i am not sure if you wanted me to change the version format as well or only increment it.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Yeah, it's correct now in its full format

**Comment by pedrobaeza** [comment]
Fast-tracking as trivial.

**Pull request merged**